README.rst

```diff
@@ -87,6 +87,20 @@ Installation
 
 The library is Python 3 only!
 
+PyPI contains binary wheels for Linux, Windows and MacOS.  If you want to install
+``yarl`` on another operation system (or *Alpine Linux* inside a Docker) the
```

Comment:
The correct word is "operating". Also, docker is irrelevant here: the containerization technique doesn't govern the presence of a compiler.
```suggestion
``yarl`` on another operating system (like *Alpine Linux*, which is not
manylinux-compliant because of the missing glibc and therefore, cannot be
used with our wheels) the
```